### DISCUSSION TITLE
(Sub-)Domains not working

### DISCUSSION THREAD

**Discussion by SaschaValentin** [Dokploy/dokploy#383] [Q&A]
I´ve installed Dokploy latest version 0.7.2 on a VPS and configured it as shown in the docs. Also pointed the domain "dokploy.mediacomvalentin.de" to the server settings and this works fine.
But as soon as I deploy an application and want to give it a subdomain like appsmith.mediacomvalentin.de this fails with an 404 error page not found.
On the other hand the automatic generated domains like test-excalidraw-ff3559-5-189-147-230.traefik.me work.
But when I go to Dokploys setting to reload Treafik an error appears with unable to reload treafik.

Is there something wrong with my settings or what can I do to solve this problem?

**Comment by Siumauricio** [comment]
Hi, Where did you have your domain? on cloudflare?

**Comment by SaschaValentin** [comment]
Hi Mauricio,

no, it is registered on my hosting provider Webgo.

But as I said: The domain for Dokploy itself works fine.

27.08.2024 21:57:25 Mauricio Siu ***@***.***>:

**Comment by missuo** [comment]
same issue

**Comment by Siumauricio** [comment]
Can yall check the traefik logs, you probably can see the root of the issue go to settings -> traefik -> show-logs, in most cases restarting traefik will work

**Comment by faizmohani** [comment]
Same issue. I want to run SAAS multi-tenancy subdomain. Basically wildcard domain (*.domain.com) . Can you help us with Traefik config?

**Comment by Siumauricio** [comment]
If anyone can give me a step by step how to reproduce I can take a look
